.gitignore

```diff
@@ -45,4 +45,4 @@ reproductions/*
 
 dev.db
 junit.xml
-/output.txt
\ No newline at end of file
+/output.txt
```

Comment:
```suggestion
/output.txt
server.key
server.csr
server.crt
root_ca.srl
root_ca.key
root_ca.crt
```
<img width="494" alt="Screenshot 2022-08-31 at 09 01 29" src="https://user-images.githubusercontent.com/35999252/187625079-6a1cc131-cd1e-4caf-93bb-f08069d88a61.png">

Let's exclude these, I am getting all these diffs when running the test suite locally

Reply:
This doesn't sound right, they shouldn't be there.

Reply:
Mini-proxy used to live in `helpers` before it was published on npm on Friday, but I can see no way these files could end up there now. Could you check if they still get generated there if you remove them?

Reply:
Also try

```
git fetch
git reset --hard origin/data-proxy-tests
```

before doing that since the branch got force pushed quite a few times

Reply:
No problems now, was my branch out of sync.